package/yast2-rear.spec

```diff
@@ -17,7 +17,7 @@
 
 
 Name:           yast2-rear
-Version:        4.2.2
+Version:        4.2.3
```

Comment:
Versioning is as defined at https://yastgithubio.readthedocs.io/en/latest/versioning/ should be 4.3.0 as target for this change is master branch, so SLE15 SP3.

Reply:
My bad, this fix should go at least in SLES15-SP2, not sure about earlier versions. The pull request has been retargeted to the SLE-15-SP2 branch so it can be merged there first. This branch does not have yet the CI pipeline so the second patch is dropped from the review.

A separate pull request will be then prepared to do a merge to the master branch, together with the update to the CI pipeline.

The patch now bumps the version from 4.2.1 to 4.2.2, but it might be better to wait for #24 to be merged first and then have this patch use 4.2.3.

Reply:
I've missed that the SLE-15-SP2 branch has a Travis pipeline, which actually fails with the current patch, not sure yet why.

Reply:
well, for master we are using github actions. for SP2 we use travis, but it does not have priority anymore.